pypit/arproc.py

```diff
@@ -604,62 +637,124 @@ def get_ampscale(slf, det, msflat):
 
 def get_datasec_trimmed(slf, fitsdict, det, scidx):
     """
-     Generate a frame that identifies each pixel to an amplifier, and then trim it to the data sections.
-     This frame can be used to later identify which trimmed pixels correspond to which amplifier
+    Primarily a wrapper with calls to get_datasec and pix_to_amp()
 
     Parameters
     ----------
-    slf : class
-      An instance of the ScienceExposure class
-    fitsdict : dict
-      Contains relevant information from fits header files
+    slf
+    fitsdict
     det : int
-      Detector number, starts at 1
     scidx : int
-      Index of science frame
 
     Returns
     -------
-    fitsdict : dict
-      Updates to the input fitsdict
+    Fills slf._datasect
+    fitsdict['naxis0'] and fittsdict['naxis1']
     """
     dnum = settings.get_dnum(det)
+    spectrograph = settings.argflag['run']['spectrograph']
+    scifile = fitsdict['directory'][scidx]+fitsdict['filename'][scidx]
```

Comment:
I don't know if this typical of how it's done so far in pypit, but with this kind of thing I tend to use `os.path.join` because it makes sure all the necessary '/' are present.  Nit-picking ...

Reply:
I prefer that too.  Edited in this case (but there are
may places that will need future editing).